README.md

```diff
@@ -367,6 +367,30 @@ module.exports = {
 
 Any `node_modules` passed into this option will be compiled through webpack as if they are part of your app.
 
+### Building a Library
```

Comment:
Thoughts on moving this to an actual docs file?

Reply:
Yeah, we should, but I think we'll do this as part of a broader docs pass.